### DISCUSSION TITLE
Adding a time offset for transcription

### DISCUSSION THREAD

**Discussion by BenBrewerBowman** [deepgram/community#471] [Feature Requests]
For my use case, I'm wanting to transcribe up to 14 hours of audio content. This is actually too much for Deepgram to handle (size is 1.1GB, but the endpoint never resolves). I am planning on having to split audio files into clips and then stitch the transcript together at the end. 

I can split them evenly at 5 hours and it will work as expected. However, when I submit each through deepgram, the timestamps won't match the original audio lenght, but each individual clip. What would be nice is to add an offset to the options when transcribing so that all video timestamps are offset by preset amount. 

For example:

```
    const transcription = await deepgramClient.transcription.preRecorded(
      { url: audioUrl },
      {
        model: "nova-2",
        language: "en-US",
        smart_format: true,
        punctuate: true,
        paragraphs: true,
        offset: 5 * 60 * 60 // 5 hours
      }
    );
```

Thoughts? Open to a better solution as well.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
